src/redux/reducers/formReducer.ts

```diff
@@ -0,0 +1,14 @@
+import { UPDATE_RESOURCE_DESCRIPTION_FORM_DATA, FormActions } from '../actions/formActions';
+import { Forms, emptyForms } from '../../types/form.types';
+
+export const formReducer = (state: Forms = emptyForms, action: FormActions) => {
```

Comment:
er det en reducer for et form eller mange forms? spør siden det brukes formReducer (entall) og state har flertall

Reply:
Jeg tenker det blir noe lignende som vi har for `errors`, hvor vi har en `forms` key som består av et objekt med x antall forms. Droppet trailing "s" siden `validationReducer` også er entall, selv om den holder styr på mange ulike errors. Det mest naturlig er kanskje å bruke konsekvent flertall, ja